vertx-pg-client/src/test/java/io/vertx/pgclient/PgConnectionTestBase.java

```diff
@@ -462,4 +463,22 @@ public void testTransactionAbort(TestContext ctx) {
       });
     }));
   }
+
+  @Test
+  public void testCloseConnectionFromDifferentContext(TestContext ctx) {
+    Async done = ctx.async(1);
+    connector.accept(ctx.asyncAssertSuccess(conn -> {
+      conn.query("SELECT 1").execute(ctx.asyncAssertSuccess(res -> {
+        ctx.assertEquals(1, res.size());
+        // schedule from another context
```

Comment:
rather use a Java thread it will create another context/event loop

i.e

new Thread(() -> ...) {
}.start();

Reply:
fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/848/commits/ce73aae89f5b511e222bceb2c6d7650fa164d448.